sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -4103,7 +4103,7 @@ object SQLConf {
       .stringConf
       .transform(_.toUpperCase(Locale.ROOT))
       .checkValues(LegacyBehaviorPolicy.values.map(_.toString))
-      .createWithDefault(LegacyBehaviorPolicy.EXCEPTION.toString)
+      .createWithDefault(LegacyBehaviorPolicy.CORRECTED.toString)
```

Comment:
> written by legacy Spark versions, instead of by third-party engines.

Not sure that reading of ancient files of Spark <= 2.4.5 is more common case then other systems. BTW, it was released almost 4 years ago.

Reply:
The files are always there unless people manually clean them up.

Reply:
Sure, but the files produces by other systems are always there too. And the amount of such files is going up faster I think.

Reply:
The question is which one is more important. This is a behavior change for existing Spark users, who might never read third-party parquet files. I don't think there is much value of this change, as it only affects the very few legacy parquet files.

On the other hand, I totally agree with flipping the default value for write-side rebase configs. There shouldn't be people using very old Spark versions to read parquet files.

Reply:
Oh nvm, the default is EXCEPTION. Then this change is fine.